tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ClientCredentialWithCertTest.cs

```diff
@@ -22,34 +22,47 @@ namespace Microsoft.Identity.Test.Unit
 {
     [TestClass]
     [DeploymentItem(@"Resources\valid_cert.pfx")]
-    public class JsonWebTokenTests
+    public class ConfidentialClientWithCertTests
```

Comment:
One of these tests is misleading because it uses ClientCredentials but the mocks return a normal (user mode) token, including idToken etc. Small refactoring in addition to fix.